### PR TITLE
feat: compatible with local GreptimeDB

### PR DESCRIPTION
**Pull request by nicecui**

### CODE REVIEW

File: start/main.py [hunk 1]
```diff
@@ -19,15 +19,20 @@ def main():
                         prog='greptime-cloud-quick-start-python',
                         description='Quick start Python demo for greptime cloud')
 
-    parser.add_argument('-host', required=True, help='The host address of the GreptimeCloud service')
-    parser.add_argument('-db', '--database', required=True, help='The database of the GreptimeCloud service')
-    parser.add_argument('-u', '--username', required=True, help='The username of the database')
-    parser.add_argument('-p', '--password', required=True, help='The password of the database')
+    parser.add_argument('-host', default="localhost", help='The host address of the GreptimeDB')
```
Comment (sunng87) [inline start/main.py:22]: tip: use `-h` for consistency with other options

File: start/main.py [hunk 2]
```diff
@@ -19,15 +19,20 @@ def main():
...
+    parser.add_argument('-db', '--database', default="public", help='The database of the GreptimeDB')
+    parser.add_argument('-u', '--username', help='The username of the database')
+    parser.add_argument('-p', '--password', help='The password of the database')
+    parser.add_argument('-P', '--port', help='The port of the HTTP endpoint of GreptimeDB')
+    parser.add_argument('--security', action=argparse.BooleanOptionalAction, default=True, help='Whether to use secure connection to GreptimeDB')
```
Comment (sunng87) [inline start/main.py]: --secure

File: start/main.py [hunk 3]
```diff
@@ -37,10 +42,22 @@ def main():
...
+        protocol = "http://"
+    
+    url = ""
+    if port:
+        url = f"{protocol}{host}:{port}"
+    else:
+        url = f"{protocol}{host}"
+    
+    url = url + f"/v1/otlp/v1/metrics"
+    print(url)
```
Comment (sunng87) [inline start/main.py]: dont print

File: start/main.py (hunk 1 above)
Comment (nicecui) [reply start/main.py:22]: `-h` is a keyword 🥲

**Review by sunng87 — APPROVED** [review]

**Pull request merged**